### PR TITLE
EPMRPP-41163 || Unique integration name impl

### PR DESCRIPTION
**Pull request by Yumfriez**

### CODE REVIEW

File: src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java [hunk 1]
```diff
@@ -79,6 +83,11 @@ public EntryCreatedRS createGlobalIntegration(IntegrationRQ createRequest, Strin
 		IntegrationService integrationService = integrationServiceMapping.getOrDefault(integrationType.getName(),
 				this.basicIntegrationService
 		);
+
+		ofNullable(createRequest.getName()).map(String::toLowerCase).ifPresent(name -> {
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java]: should we do something if name is null ?

File: src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java [hunk 2]
```diff
@@ -102,6 +111,11 @@ public EntryCreatedRS createProjectIntegration(String projectName, IntegrationRQ
 				this.basicIntegrationService
 		);
 
+		ofNullable(createRequest.getName()).map(String::toLowerCase).ifPresent(name -> {
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java]: should we do something if name is null ?

File: src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java (hunk 1 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java]: validation added

File: src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java (hunk 2 above)
Comment (Yumfriez) [reply src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java]: validation added

File: src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java [hunk 3]
```diff
@@ -79,6 +84,13 @@ public EntryCreatedRS createGlobalIntegration(IntegrationRQ createRequest, Strin
 		IntegrationService integrationService = integrationServiceMapping.getOrDefault(integrationType.getName(),
 				this.basicIntegrationService
 		);
+
+		String integrationName = ofNullable(createRequest.getName()).map(String::toLowerCase).map(name -> {
```
Comment (pbortnik) [inline src/main/java/com/epam/ta/reportportal/core/integration/impl/CreateIntegrationHandlerImpl.java]: what about annotations @NotNull and @NotEmpty for fields in IntegrationRQ?

**Pull request merged**